permits/api.py

```diff
@@ -173,6 +154,18 @@ def has_permission(self, request, view):
             return False
 
 
+class BlockRequesterUserWithoutGroup(BasePermission):
```

Comment:
Adapt all endpoints access according to the whiteboard on SIT room

Reply:
I don't see the whiteboard :-)